projects/commudle-admin/src/app/components/navbar-menu/navbar-menu.component.ts

```diff
@@ -1,22 +1,64 @@
-import { Component, Input, OnInit } from '@angular/core';
-import { faFlask, faHome, faInfoCircle, faLightbulb, faUserFriends } from '@fortawesome/free-solid-svg-icons';
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { faFlask, faHome, faInfoCircle, faLightbulb, faUserFriends, faBell } from '@fortawesome/free-solid-svg-icons';
 import { ICurrentUser } from 'projects/shared-models/current_user.model';
+import { NotificationsPopoverComponent } from '../../feature-modules/notifications/components/notifications-popover/notifications-popover.component';
+import { NotificationChannel } from 'projects/commudle-admin/src/app/feature-modules/notifications/services/websockets/notification-channel';
 
 @Component({
   selector: 'app-navbar-menu',
   templateUrl: './navbar-menu.component.html',
   styleUrls: ['./navbar-menu.component.scss'],
 })
-export class NavbarMenuComponent implements OnInit {
+export class NavbarMenuComponent implements OnInit, OnDestroy {
   @Input() currentUser: ICurrentUser;
 
   faHome = faHome;
   faInfoCircle = faInfoCircle;
   faLightbulb = faLightbulb;
   faFlask = faFlask;
   faUserFriends = faUserFriends;
+  faBell = faBell;
 
-  constructor() {}
+  mobileView: boolean;
+  notificationCount = 0;
 
-  ngOnInit(): void {}
+  notificationsPopoverComponent = NotificationsPopoverComponent;
+
+  subscriptions = [];
+
+  constructor(private notificationChannel: NotificationChannel) {}
+
+  ngOnInit(): void {
+    if (/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)) {
```

Comment:
Why is this needed?

Reply:
To differentiate between desktop and mobile view.